src/desktop/handle_token.rs

```diff
@@ -5,7 +5,10 @@ use std::{
 
 use rand::{distributions::Alphanumeric, thread_rng, Rng};
 use serde::{Deserialize, Serialize};
-use zbus::{names::OwnedMemberName, zvariant::Type};
+use zbus::{
+    names::OwnedMemberName,
+    zvariant::{OwnedObjectPath, Type},
```

Comment:
This is only usef if the backend feature is enabled, causing a conditional clippy warning